Gemfile

```diff
@@ -6,10 +6,11 @@ gem 'pry'
 gem 'pry-byebug'
 gem 'rake', '12.3.0'
 gem 'rspec', '~> 3.6'
+gem 'rubocop'
```

Comment:
It's recommended to use `require: false` for Robocop in Gemfile
https://github.com/rubocop-hq/rubocop#installation